.travis.yml

```diff
@@ -27,19 +27,19 @@ jobs:
         - sudo mv docker-compose /usr/local/bin
         - sudo service mysql stop
       install:
-        - python3 codalab_service.py build services --version ${TRAVIS_BRANCH} --pull $([ -z "${CODALAB_DOCKER_USERNAME}" ] || [ "$TRAVIS_PULL_REQUEST_BRANCH" != "" ] || echo "--push")
-        - python3 codalab_service.py start --services default monitor --version ${TRAVIS_BRANCH}
+        - python3 codalab_service.py build services --pull $([ -z "${CODALAB_DOCKER_USERNAME}" ] || [ "$TRAVIS_PULL_REQUEST_BRANCH" != "" ] || echo "--push")
+        - python3 codalab_service.py start --services default monitor
         - pip install -r requirements.txt
       before_script:
         - ./scripts/test-setup.sh
         - export PATH=$PATH:/usr/bin/geckodriver
       script:
-        - python3 codalab_service.py test --version ${TRAVIS_BRANCH} default
+        - python3 codalab_service.py test default
```

Comment:
@percyliang @candicegjing @yipenghe 

Note that I've updated Travis not to directly pass in the branch name through the `--version` arg. This is because if the `--version` tag is not provided, then `get_default_version()` is already called, which already gets the branch name -- so using the `--version` tag is not necessary.

Additionally, `get_default_version()` automatically replaces the `/` and `-` characters in branch names, allowing branches such as this one (`dependabot/pip/psutil-5.6.6`) to be built.

Reply:
Is the git branch identical to `$TRAVIS_BRANCH`?

Reply:
Just checked and, actually, running `get_default_version()` on Travis actually always returns `HEAD` rather than the branch name because the branch is checked out in a "detached HEAD" state (see https://unhashable.com/getting-the-current-branch-name-during-a-pull-request-in-travis-ci/)

So I'lll change this PR so it uses the `--version` argument as before, but just do the string substitution through Python.